### PR TITLE
[#481] automate the part to grep the explorer ips

### PR DESCRIPTION
**Pull request by jhd2best**

### CODE REVIEW

File: devops/create_https_wss_endpoints_aws/helpers.py [hunk 1]
```diff
@@ -16,13 +16,59 @@
...
+    network_config_cache = network_config_dict
+
+    # analyze network type
+    domain_name = network_config_dict['domain_name']
+    domain_name_parts = domain_name.split('.')
+    network_abbr = domain_name_parts[0]
+    return network_abbr
+
+
+def parse_distribution_config(file_path):
```
Comment (AndyBoWu) [inline devops/create_https_wss_endpoints_aws/helpers.py:36]: kindof overkill to create a dict and method for this task imo. We just need to `grep` the explorer IP from the `distribution_config.txt` file and plug them in the main method.

Comment (AndyBoWu) [reply devops/create_https_wss_endpoints_aws/helpers.py:36]: This also further simplify the master template.

**Comment by AndyBoWu** [comment]
This is a good implementation, but kind of overkill in general for this task. Let me refactor the code a bit, and share the result with you. @jhd2best

**Comment by LeoHChen** [comment]
Is it still needed?

**Comment by jhd2best** [comment]
seems no use later. let me close it

**Pull request closed without merging**